debian/control

```diff
@@ -0,0 +1,41 @@
+Source: freedom-maker
+Maintainer: Joseph Nuthalapati <njoseph@riseup.net>
+Uploaders:
+ Ferderico Ceratto <federico3@debian.org>
+Section: python
+Priority: optional
+Build-Depends:
+ debhelper (>= 9),
+ dh-python,
+ dh-systemd,
+ python3-all
+Standards-Version: 3.9.8
+Homepage: https://freedombox.org
+Vcs-Browser: https://anonscm.debian.org/git/freedombox/freedom-maker.git
+Vcs-Git: https://anonscm.debian.org/git/freedombox/freedom-maker.git
+
+Package: freedom-maker
+Architecture: all
+Depends:
+ ${misc:Depends},
+ ${python3:Depends},
+ binfmt-support,
+ btrfs-tools,
+ dosfstools,
+ extlinux,
+ git,
+ pxz,
+ qemu-user-static,
+ qemu-utils,
+ sshpass,
+ sudo,
+ u-boot-tools,
+ virtualbox,
```

Comment:
Note that virtualbox is only in Debian contrib (and only in unstable). I think it can only be listed as "Suggests".